### PR TITLE
cookie policy component

### PR DESCRIPTION
**Pull request by dejanajan**

### CODE REVIEW

File: src/components/_molecules/CookiePolicy/CookiePolicy.component.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import Button from '../../_atoms/Button';
+
+export type CookiePolicyProps = {
+  title?: string;
+  text?: React.ReactNode;
+  buttonContentF?: React.ReactNode;
+  buttonContentS?: React.ReactNode;
+  onHandleDecline?: () => void;
+  onHandleAccept?: () => void;
+};
```
Comment (deltafixer) [inline src/components/_molecules/CookiePolicy/CookiePolicy.component.tsx:13]: Can we cleaner names? I.e. 'acceptButton', 'declineButton'. We can also use just one callback prop (i.e. name it 'userAction') and let it return either accepted or declined action

File: src/components/_molecules/CookiePolicy/CookiePolicy.component.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    text,
+    buttonContentF,
+    buttonContentS,
+    onHandleDecline,
+    onHandleAccept,
+  } = props;
+
+  const [isShowing, setIsShowing] = useState(true);
+
+  function deleteCookies() {
```
Comment (deltafixer) [inline src/components/_molecules/CookiePolicy/CookiePolicy.component.tsx:29]: const deleteCookies = () => ...

File: src/components/_molecules/CookiePolicy/CookiePolicy.component.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  return (
+    <Styled.Container>
+      {isShowing ? (
+        <div className="cookie-policy">
+          <Text.Heading level="3" className="cookie-policy__title">
+            {title}
+          </Text.Heading>
+          <Text.Body className="cookie-policy__text">{text}</Text.Body>
+          <Button className="cookie-policy__btnF" onClick={acceptCookies}>
```
Comment (deltafixer) [inline src/components/_molecules/CookiePolicy/CookiePolicy.component.tsx:68]: also, className here, "btnF", can be named __accept-button

File: src/components/_molecules/CookiePolicy/CookiePolicy.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import CookiePolicy from '.';
+
+test('CookiePolicy renders correctly', () => {
+  const component = renderer.create(
+    <CookiePolicy
+      title={'Cookie Policy'}
+      text={`Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s.`}
+      buttonContentF={<p>Accept</p>}
+      buttonContentS={<p>Reject</p>}
+    />
```
Comment (deltafixer) [inline src/components/_molecules/CookiePolicy/CookiePolicy.spec.tsx:12]: Don't forget to update spec after updating the component

File: src/components/_molecules/CookiePolicy/CookiePolicy.component.tsx (hunk 1 above)
Comment (deltafixer) [reply src/components/_molecules/CookiePolicy/CookiePolicy.component.tsx:13]: If it's always wrapped in <Button>...</Button> then props can be named 'acceptButtonContent' and 'declineButtonContent'. Or, if the plan is to pass a whole piece of code, then we don't need the <Button/> in this component's code, we'll pass the 'pre-created' (already wrapped) button to it.

File: src/components/_molecules/CookiePolicy/CookiePolicy.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+import Button from '../../_atoms/Button';
+import Icon from '../../_atoms/Icon';
+import Text from '../../_atoms/Text';
+
+import CookiePolicy from "./CookiePolicy.component.tsx";
+
+<Meta title="Core|Organisms/CookiePolicy" />
+
```
Comment (deltafixer) [inline src/components/_molecules/CookiePolicy/CookiePolicy.stories.mdx:9]: Is this an Organism or a molecule :D

File: src/components/_molecules/CookiePolicy/CookiePolicy.styles.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+      margin-bottom: ${pxToRem('20px')};
+      color: #fff;
+    }
+
+    &__btnF,
+    &__btnS {
+      display: inline-flex;
+      justify-content: space-between;
+      flex-direction: row;
+      align-items: center;
```
Comment (deltafixer) [inline src/components/_molecules/CookiePolicy/CookiePolicy.styles.tsx:21]: we have the `flexRow` mixin, you can use it here

File: src/components/_molecules/CookiePolicy/CookiePolicy.styles.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import { pxToRem, padding } from '@faxi/web-css-utilities';
+
+export const Container = styled.div`
+  .cookie-policy {
+    ${padding(pxToRem('20px'), pxToRem('50px'))};
+    background-color: #00708d;
```
Comment (deltafixer) [inline src/components/_molecules/CookiePolicy/CookiePolicy.styles.tsx:7]: We shouldn't have colors in WCL components. Colors should be passed as props or just propagated to their parents and passed via `className`. Check out how other components in this library have colors.

File: src/components/_molecules/CookiePolicy/index.ts [hunk 8]
```diff
@@ -0,0 +1,3 @@
+import CookiePolicy from './CookiePolicy.component';
+
+export default CookiePolicy;
```
Comment (deltafixer) [inline src/components/_molecules/CookiePolicy/index.ts:3]: CookiePolicy should also be exported from `src/index.ts`

**Comment by Jonjoe** [comment]
@deltafixer  has called out most of the points. Once they are cleared we can re-review :)

**Pull request merged**